requirements/testing.txt

```diff
@@ -1,4 +1,4 @@
-coverage
+coverage[toml]
```

Comment:
Is this how we install extras? Let's see what the test suite says. It seems to work locally, but that's with Python 3.11...

Reply:
https://github.com/django-crispy-forms/django-crispy-forms/actions/runs/3813140645/jobs/6486715584#step:6:42

Coverage continues to work all the way back to Python 3.7, with branch reporting.